docs/extensions.md

```diff
@@ -102,3 +102,22 @@ module.exports = robot => {
 ```
 
 Check out [stale](https://github.com/probot/stale) to see it in action.
+
+## Attachments
+
+[probot-attachments](https://github.com/probot/attachments) adds message attachments to comments on GitHub. This extension should be used any time an app is appending content to user comments.
```

Comment:
> This extension should be used any time an app is appending content to user comments.

I'd love to figure out a way to emphasize this. In general, I feel like bots should not be encouraged to modify comments. There are a few exceptions (e.g. censorship), but otherwise I feel like they should only "append" content.

Reply:
We could add a note at the bottom like:

"Please only use this extension when absolutely necessary. In most use cases, apps should act independently and not directly edit user's content."